testutils/check-baselines.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+# TODO: We should figure out how to do this as part of GapicGeneratorTest at some point.
+
+exitStatus=0
+
+## Check for unhandled cases.
+unhandledStr='$unhandledCallingForm'
+if grep --include='*.baseline' -r -l -F $unhandledStr src/; then
+	cat 1>&2 <<EOF
+Above baseline files contain string "$unhandledStr";
+some sample code won't render properly.
+
+EOF
+	exitStatus=1
+fi
+
+## Check that every calling form is used somewhere.
+callingFormsFile="src/main/java/com/google/api/codegen/viewmodel/CallingForm.java"
+
+# Calling forms are in an enum, so we look for capital identifiers that begins the line.
+callingForms=$(< $callingFormsFile grep -o '^[[:space:]]*[A-Z][a-zA-Z]*' | tr -d -c '[a-zA-Z\n]')
+
+unusedForms=$(
+	# Grep src/
+	#   recursively (-r)
+	#   in baseline files (--include='*.baseline')
+	#   for any calling forms: -f <(cat <<< $callingForms)
```

Comment:
The use of the here string is neat, but `-f <(echo "$callingForms")` accomplishes the same thing, right?

Reply:
This is slightly subtle but it does not. Documented this into the script.